serving/docs/lmi/tutorials/seq_scheduler_tutorial.md

```diff
@@ -0,0 +1,99 @@
+#  Seq-Scheduler Tutorial
+
+Seq-scheduler is a language model serving system built on native Huggingface inference backend, and works with any 
+huggingface model out-of-box. It features dynamic batching, dynamic batch splitting and prompt caching, with the 
+support of various autoregressive search methods, including greedy, sampling and [contrastive search](https://huggingface.co/blog/introducing-csearch). In the 
+following part of the tutorial, the way to run these features will be introduced.
+
+## Start a model server with a serving.properties file
+The model server is configured by a stand-along serving.properties file. Here `/workspace/llama-2-70b-hf/` model is 
+used as an example. 
+```
+# serving.properties
+
+engine=MPI
+option.model_id=/workspace/llama-2-70b-hf/
+option.tensor_parallel_degree=1
+option.dtype=fp16
+option.trust_remote_code=true
+
+# rolling-batch parameters
+option.max_rolling_batch_size=64
+option.rolling_batch=scheduler
+
+# seq-scheduler parameters
+option.max_sparsity=0.33
+option.max_splits=3
+option.decoding_strategy=contrastive  # other options: greedy, sample
+```
+
+Then inside a djl-serving docker image, by running the following command, the server is started.
+```bash
+djl-serving -m <directory-of-serving.properties>
+```
```

Comment:
Why should it be inside a djl-serving? Users can use this serving.properties in docker run or in SageMaker right?

Reply:
No these will be added in DJL docs which will be shared with our customers. bashing into a docker image and running it, is un-neccessary and confusing for users. Here is another tutorial, we just published now https://docs.djl.ai/docs/serving/serving/docs/lmi/tutorials/trtllm_aot_tutorial.html